dds_web/templates/user/confirm2fa.html

```diff
@@ -0,0 +1,47 @@
+{% extends "bootstrap/base.html" %}
+{% import "bootstrap/wtf.html" as wtf %}
+
+{% block content %}
+<title>2FA</title>
+
+<body>
+    <h1>Enter one-time authentication code</h1>
+    {% with messages = get_flashed_messages(with_categories=true) %}
+    {% if messages %}
+    <div class=flashes>
+        {% for category, message in messages %}
+        <div class="alert alert-{{category}}" role="alert">
+            {{ message }}
+        </div>
+        {% endfor %}
+    </div>
+    {% endif %}
+    {% endwith %}
+    <p>Please finish the login authenticate yourself to trigger the sending of an authentication one-time code.
+        The one-time codes are valid for a short time (1 hour) after they have been requested.
```

Comment:
This sentence needs a rephrase and we should change 1 hour to 15 minutes both in text and code logic.

Reply:
That's a very strange sentence I agree. Ok, let's go with 15 min then.

Reply:
Should be adjusted now.